runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCacheCursorFactory.java

```diff
@@ -71,7 +73,7 @@ public KafkaCacheCursorFactory(
     {
         this.writeBuffer = writeBuffer;
         this.checksum = new CRC32C();
-        this.nullKeyInfo = initNullKeyInfo(checksum);
+        this.nullKeyRO = initNullKeyRO();
```

Comment:
We were previously using this approach to avoid copying the `null` key value for each client filter.
I don't see where we are using `nullKeyRO` after the changes.